content/en/docs/tasks/extend-kubectl/kubectl-plugins.md

```diff
@@ -43,6 +50,11 @@ Executing this command causes a traversal of all files in your PATH. Any files t
 A warning will be included for any files beginning with `kubectl-` that are *not* executable.
 A warning will also be included for any valid plugin files that overlap each other's name.
 
+To discover plugins that are available in the open source, you can use
+[Krew](https://sigs.k8s.io/krew) which maintains an open source [plugin
```

Comment:
hello @corneliusweig. My original comment was more basic -- the phrase `in the open source` read differently than I thought it should. In general, the PR seems fine.
I _think_ the docs are in code freeze now.
/cc @simplytunde

Reply:
How about:
```markdown
You can use [Krew](https://sigs.k8s.io/krew) to discover and select from a
community-curated set of `kubectl` plugins.
```

Reply:
@kbhawkey I agree that the discussion about plugin licenses should move elsewhere. I therefore like @sftim 's suggestion, because it is less specific about the licensing :+1: